### PR TITLE
[FE] When navigating into screen with list of surveys, automaticaly click on first if there is only one

### PR DESCRIPTION
**Pull request by shirve**

### CODE REVIEW

**Review by tobob — APPROVED** [review]

File: team-fingerprints-web/src/components/SurveyList/SurveyList.tsx [hunk 1]
```diff
@@ -85,6 +87,10 @@ const SurveyList = () => {
       survey: el,
     }));
 
+    if (mappedData?.length === 1) {
```
Comment (tobob) [inline team-fingerprints-web/src/components/SurveyList/SurveyList.tsx:90]: please move this into useEffect with additional check if it finish loading
please don't place such code inside useMemo

**Review by tobob — CHANGES_REQUESTED** [review]

**Review by tobob — APPROVED** [review]

**Comment by tobob** [comment]
@shirve please resolve conflicts

**Comment by Matesty** [comment]
@shirve
- Back button from browser and inside the app doesn't work, once you open the survey
- User should be able to see survey list after clicking 'Survey List' from breadcrumbs (?)
- Should it work the same for own surveys?

Zrzut ekranu_20221128_102905

**Review by tobob — APPROVED** [review]

**Comment by Matesty** [comment]
@shirve Works good, but still not for own surveys

**Comment by shirve** [comment]
> @shirve Works good, but still not for own surveys

About that - if there is only one survey in `Own Surveys` and it has not yet been filled do we want to redirect user to it too?

**Comment by shirve** [comment]
@Matesty you can test it now with Own Surveys

**Review by Matesty — APPROVED** [review]

**Review by tobob — APPROVED** [review]

**Pull request merged**